dqn.cpp

```diff
@@ -173,19 +182,19 @@ void DQN::Update() {
     assert(!std::isnan(target));
     target_input[i * kOutputCount + static_cast<int>(action)] = target;
     filter_input[i * kOutputCount + static_cast<int>(action)] = 1;
-    for (auto j = 0; j < kInputCount; ++j) {
+    for (auto j = 0; j < kInputFrameCount; ++j) {
       const auto& frame_data = std::get<0>(transition)[j];
       std::copy(frame_data->begin(), frame_data->end(), states_input.begin() +
-                i * kInputDataSize + j * kStateDataSize);
+                i * kActorInputDataSize + j * kActorStateDataSize);
     }
   }
-  InputDataIntoLayers(*net_, states_input, target_input, filter_input);
-  solver_->Step(1);
+  InputDataIntoLayers(*actor_net_, states_input, target_input, filter_input);
+  actor_solver_->Step(1);
```

Comment:
This update is going to have be changed a lot in the future in the ways we had talked about. But for now don't worry about it.